include/m_event.h

```diff
@@ -379,6 +379,11 @@ typedef struct ghost_event_s {
 
 #define mEv_DESGINER_NUM 3
 
+typedef struct designer_common_s {
+  int button_presses;
+  int _04;
```

Comment:
Judging that this is the number of button presses based off the fact that there are checks against `100`. I wasn't sure what the second argument was since I think I need to reverse engineer the designer event TU next to know for sure probably.